spec/models/rent_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe Rent, type: :model do
+  it do
+    should validate_presence_of(:user)
+  end
+
+  it do
+    should validate_presence_of(:book)
+  end
+
+  subject(:rent) do
```

Comment:
Usually when it does not span more than one line, you can put this in one line like this:
`subject(:rent) { build(:rent) }`
This applies to `let` blocks too.

Reply:
Done, thanks!